azure-pipelines.yml

```diff
@@ -84,14 +84,15 @@ jobs:
       - checkout: self
       - bash: |
           set -euo pipefail
-          eval "$(./dev-env/bin/dade-assist)"
```

Comment:
I guess this is just redundant since you only need `git`?

Reply:
Yes; it also doesn't work on either macOS or Windows.